src/app/layout/pages/main-dashboard-page/main-dashboard-page.component.ts

```diff
@@ -35,7 +35,7 @@ export class MainDashboardPageComponent {
     })
 
     effect(() => {
-      this._loggedOut()
+      if (!this._loggedOut()) return
       void this._router.navigate(['auth/login'])
```

Comment:
Your code will never reach this line because you return on the previous. Is that ok?